packages/rest/src/lib/handlers/SequentialHandler.ts

```diff
@@ -391,7 +391,7 @@ export class SequentialHandler implements IHandler {
 			}
 		}
 
-		if (status === 200) {
+		if (status === 200 || status === 201) {
```

Comment:
For future proofing we should accept anything within the ok range, like we did before.

```suggestion
		if (status >= 200 && status <= 299) {
```

Reply:
sounds good since it previously relied on Response.ok which refers to a 2xx code.
to match the formatting of other status checks, seems like it should be a `< 300` though?

Reply:
Yeah I've updated my diff.